ckan/lib/webassets_tools.py

```diff
@@ -127,6 +127,7 @@ def include_asset(name: str) -> None:
 
     bundle: Any = env[name]
     deps: list[str] = bundle.extra.get("preload", [])
+    attrs: dict[str, str] = bundle.extra.get("attrs", {})
```

Comment:
yaml could include almost any type of object. Force str key and values here or raise an exception if the types are wrong?